### PR TITLE
Python RL Environment

### PR DESCRIPTION
**Pull request by fjones36**
Install the requirements.

Make the CLI.

Ray has a dashboard that gives you some performance statistics. When using WSL 2 you have to set the dashboard host to the IP of your WSL VM, so if you look in train.py, you can see in ray.init() where to set the address. If you comment it out it defaults to localhost.

All training runs get logged to ~/ray_results automatically.

Run: python train.py

This will train a DQN policy for player 1 and create a random policy for player 2.

You can view the ray dashboard, but it might be easier to just load up tensorboard directly to look at training performance:
tensorboard --logdir ~/ray_results

### CODE REVIEW

**Comment by codpro880** [comment]
Looks good! I think my localhost issues are personal problems (I left some deets in slack). Minimally I can see the policy/avg-reward mostly getting better over time

File: src/python/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+gym
+ray[rllib]==1.2
```
Comment (aecallahan) [inline src/python/requirements.txt:2]: Are these the only required packages?

Comment (codpro880) [reply src/python/requirements.txt:2]: This is what I have if you want to run the dashboard on localhost:

```
(bgsrl) Matthews-MacBook-Pro:python matt$ cat requirements.txt 
gym
ray[rllib]==1.2
#ray[tune]
pandas
tabulate
tensorboard
torch
```

Comment (fjones36) [reply src/python/requirements.txt:2]: Sorry, 
ya that looks about right. I think I only import gym and ray, but I had to also install tensorboard and pytorch (which I forgot). I'm also using a conda environment so pandas and tabulate probably came with it.

**Pull request merged**